travisci/add_service.xml

```diff
@@ -0,0 +1,409 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
Move this to dockstore-webservice/src/main/resources like dockstore-webservice/src/main/resources/migrations.test.samepaths.xml to keep things organized

Change the name to add_service_1.7.0.xml ala our confidential migrations so we know what version this was for